src/index.js

```diff
@@ -1,15 +1,54 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
+// function component() {
+//     const element = document.createElement('div');
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script 
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
-  }
-  
-  document.body.appendChild(component());
\ No newline at end of file
+//     // Lodash, now imported by this script
+//     element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+//     element.classList.add('hello');
+
+//     return element;
+//   }
+
+//   document.body.appendChild(component());
```

Comment:
- [ ] Please remove commented code as it makes you code messy and difficult to read. Kindly make changes here. :pray:

Reply:
This issue has been addressed well. Great job! :clap: :+1: